about/release-notes.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Timescale release notes and future plans
+excerpt: New features and fixes are released regularly in all Timescale products
+keywords: [TimescaleDB, Timescale Cloud, TimescaleDB Toolkit, pgai, pgvectorscale, pgspot, live-migration, upgrades, updates, releases]
+---
+
+# Timescale release notes
+
+For information about major release for all Timescale product, see the [Changelog][changelog]. For product release 
```

Comment:
The changelog is about Timescale Cloud specifically. Let's make that clear here and in the changelog.

For example, a TimescaleDB release is included in the release notes on Github as soon as it it released but it's not published in the changelog until it is available on Timescale Cloud.

Reply:
As before. Don't want to give the impression that these products are self-hosted only.